### PR TITLE
[EDU-3781] refactor: uxw digital certificates

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Labels and placeholders for digital certificate views
- Order of form radio in create view
- Oder of form element name in edit view
- Label in domains view

### Attachments

image

image 

image

image

image

image

### CODE REVIEW

**Comment by matheus-sunderhus** [comment]
Para resolver o CLA @bru-andrade , basta escrever aqui no seu PR neste campo de comentário o texto:
I have read the CLA Document and I hereby sign the CLA

**Comment by bru-andrade** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by matheus-sunderhus — APPROVED** [review]
LGTM

**Review by matheus-sunderhus — DISMISSED** [review]

**Comment by matheus-sunderhus** [comment]
@bru-andrade , só conseguir mais 1 approve com alguém do time de writer e fazemos o merge 👍🏻

File: src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue [hunk 1]
```diff
@@ -110,15 +109,15 @@
           }"
         >
           <template #title>
-            <span class="text-base">Edge Certificate - Upload</span>
+            <span class="text-base"> Import a certificate</span>
```
Comment (hannah-sk) [inline src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue:112]:
```suggestion
            <span class="text-base">Import a certificate</span>
```

File: src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue [hunk 2]
```diff
@@ -131,15 +130,17 @@
           }"
         >
           <template #title>
-            <span class="text-base">Edge Certificate - Generate</span>
+            <span class="text-base"> Request a certificate </span>
```
Comment (hannah-sk) [inline src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue:133]:
```suggestion
            <span class="text-base">Request a certificate</span>
```

File: src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue [hunk 3]
```diff
@@ -152,24 +153,26 @@
           }"
         >
           <template #title>
-            <span class="text-base">Trusted CA Certificate</span>
+            <span class="text-base"> Import a Trusted CA certificate </span>
```
Comment (hannah-sk) [inline src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue:156]:
```suggestion
            <span class="text-base">Import a Trusted CA certificate</span>
```

File: src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue [hunk 4]
```diff
@@ -209,9 +212,8 @@
   </FormHorizontal>
 
   <FormHorizontal
-    title="Generate CSR and Private Key with Azion"
-    description="To apply for a digital certificate issued by a certificate authority, a Certificate Signing Request is required.
-        Azion can generate a certificate code to submit to a certificate authority."
+    title="Request a Certificate"
+    description="Generate a CSR and private key and submit to a certificate authority to generate a digital certificate."
```
Comment (hannah-sk) [inline src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue:216]:
```suggestion
    description="Generate a CSR and private key and submit them to a certificate authority to generate a digital certificate."
```

File: src/views/DigitalCertificates/FormFields/FormFieldsEditDigitalCertificates.vue [hunk 5]
```diff
@@ -43,9 +43,9 @@
 <template>
   <FormHorizontal
     v-if="csr"
-    title="Generate CSR and Private Key with Azion"
-    description="The Certificate Signing Request with Azion was generated. Copy the certificate and submit it to a certificate authority.
-        Paste the certificate in the respective field and click Save."
+    title=" Update CSR "
```
Comment (hannah-sk) [inline src/views/DigitalCertificates/FormFields/FormFieldsEditDigitalCertificates.vue:46]:
```suggestion
    title="Update CSR"
```

**Review by hannah-sk — APPROVED** [review]

**Review by matheus-sunderhus — APPROVED** [review]

**Pull request merged**